remove-stress.lua

```diff
@@ -8,42 +8,55 @@ remove-stress
 Sets stress to -1,000,000; the normal range is 0 to 500,000 with very stable or
 very stressed dwarves taking on negative or greater values respectively.
 Applies to the selected unit, or use ``remove-stress -all`` to apply to all units.
+Using the argument ``-value [ 0 ]`` will reduce stress to the value 0 instead of -1,000,000.
 
 ]====]
 
 local utils = require 'utils'
 
-function removeStress(unit)
+function removeStress(unit,value)
     if unit.counters.soldier_mood > df.unit.T_counters.T_soldier_mood.Enraged then
         -- Tantrum, Depressed, or Oblivious
         unit.counters.soldier_mood = df.unit.T_counters.T_soldier_mood.None
     end
     if unit.status.current_soul then
-        unit.status.current_soul.personality.stress_level = -1000000
+        if unit.status.current_soul.personality.stress_level > value then
+            unit.status.current_soul.personality.stress_level = value
+        end
     end
 end
 
 local validArgs = utils.invert({
     'help',
-    'all'
+    'all',
+    'value'
 })
 
 function main(...)
     local args = utils.processArgs({...}, validArgs)
+    local stress_value = -1000000
 
     if args.help then
         print(help)
         return
     end
+    
+    if args.value then
+        if args.value[1] then
```

Comment:
Instead of making `-value` take an array containing a single item, I would just make it take a number (`-value x`) and check for `if tonumber(args.value) ~= nil` here

Reply:
How do I handle negative values? Using `-value -1` throws an error for an invalid argument.

Reply:
[set-belief](https://github.com/DFHack/scripts/blob/47d9b3c9390d21ecc49ddaba477b16071884ee87/modtools/set-belief.lua#L44) (which uses processArgs) mentions escaping negative values with `\`, so I assume that would work (but it might actually need to be `\\` - I haven't tested)